Sources/TIM/AppAuth/AppAuthController.swift

```diff
@@ -243,5 +238,34 @@ public final class AppAuthController: OpenIDConnectController {
             completion(result)
         }
     }
+    
+    /// Create AuthorizationRequest using the authorizationRequestNonce in case it is provided
+    private func createAuthorizationRequest(config: OIDServiceConfiguration, authorizationRequestNonce: String?) -> OIDAuthorizationRequest {
+        if authorizationRequestNonce == nil {
```

Comment:
I would do `if let authorizationRequestNonce = authorizationRequestNonce { ... } else { ... }` as it's more swift to do